coretex/cli/modules/utils.py

```diff
@@ -157,6 +157,9 @@ def onBeforeCommandExecute(fun: Callable[..., Any], excludeOptions: Optional[Lis
     def decorator(f: Callable[..., Any]) -> Callable[..., Any]:
         @wraps(f)
         def wrapper(*args: Any, **kwargs: Any) -> Any:
+            if click.get_current_context().invoked_subcommand in excludeOptions:
```

Comment:
`excludedOptions` is used for excluding parametar of type option when an command is run. Can we add `excludedCommands`, or `excludedSubcommands` parameter to use in cases like these?